index.js

```diff
@@ -50,7 +57,20 @@ function getAllMovieTitles() {}
  *  checkIfAnyMovieHasRating(movies, "R");
  *  //> false
  */
-function checkIfAnyMovieHasRating() {}
+function checkIfAnyMovieHasRating(movies,rating) {
+  if(rating=== undefined){
+    rating = 'G'
+  }
```

Comment:
```suggestion
function checkIfAnyMovieHasRating(movies,rating = "G") {
```
Nice job on this one David! You can set a default value for parameters using this syntax.